src/_components/badge/Badge.svelte

```diff
@@ -20,10 +20,6 @@
   type TType = 'danger' | 'primary' | 'secondary' | 'succes';
   export let type: TType;
 
-  if (type) {
-    cssClass = `${cssClass} badge--${type}`;
-  }
-
   let style: string = '';
```

Comment:
Jeg ved godt at det nok ikke er fra denne, men kan du ikke sætte den til undefined - så slipper vi for en tom style attribut